aidial_adapter_bedrock/dial_api/request.py

```diff
@@ -44,3 +46,20 @@ def create(cls, request: ChatCompletionRequest) -> "ModelParameters":
 
     def add_stop_sequences(self, stop: List[str]) -> "ModelParameters":
         return self.copy(update={"stop": [*self.stop, *stop]})
+
+
+class EmbeddingsType(str, Enum):
+    SYMMETRIC = "symmetric"
+    DOCUMENT = "document"
+    QUERY = "query"
+
+
+class EmbeddingsRequest(ExtraAllowModel):
```

Comment:
Why do we want ExtraAllowModel here?

Because I've tried to run this API with one extra param in body

```python
{"input": ["fish", "ball"], "streaming": True}
```
And it has resulted into 500 error from InvokeModel operation itself

I don't think it's expected behaviour in such situations

Reply:
Indeed, fixed. Such a request now returns 400.